internal/cli/atlas/atlas.go

```diff
@@ -70,6 +74,7 @@ func Builder() *cobra.Command {
 
 			return validate.Credentials()
 		},
+		Deprecated: "see https://dochub.mongodb.org/core/migrate-to-atlas-cli to migrate to Atlas CLI.",
```

Comment:
This is the second change.
This string is shown only when running `mongocli atlas`

```
./bin/mongocli atlas
Command "atlas" is deprecated, see https://dochub.mongodb.org/core/migrate-to-atlas-cli to migrate to Atlas CLI.
MongoDB Atlas operations.

Usage:
  mongocli atlas [command]
```

Reply:
I had to change the message after taking with design